.schemas/authenticators.oauth2_introspection.schema.json

```diff
@@ -146,6 +146,35 @@
           }
         }
       ]
+    },
```

Comment:
You can revert these changes as this file is no longer being used and is just kept for backwards compatibility